index.html

```diff
@@ -3881,10 +3881,21 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="image">
+			<rdef>image</rdef>
+			<div class="role-description">
+				<p>A container for a collection of <a>elements</a> that form an image. See synonym <rref>img</rref>.</p>
+				<div class="note" id="role-image-note-image">
+					<h5>Note regarding the ARIA 1.3 <code>image</code> role.</h5>
+					<p>In ARIA 1.3, the working group introduced <code>image</code> as a synonym to the <rref>img</rref> role. This was done to mitigate author confusion and errors due to the role being an outlier compared to all other roles, which are complete words or concatenations of complete words.</p>
+					<p>Until implementations include sufficient support for <code>role="image"</code>, web authors are advised to use both roles as a fallback <code>role="image img"</code> or to use the <code>img</code> role alone.</p>
```

Comment:
I thought we have previously determined that fallback roles do not solve backward compatibility problems.

Text already states that this is an ARIA 1.3 addition, so isn't that sufficient? Perhaps we do not need this paragraph.

Reply:
i had thought we had talked about that during a call as well, but james craig added it in so I assumed I was mistaken?

Reply:
We removed from the example on none once it was implemented in all modern user agents. The only one lacking was IE which did not support role fallbacks anyway.

Probably best to add here - although hopefully we end up being able to remove this some time within the development cycle of 1.3 if it is adequately supported.